airflow/www/static/js/dag/details/taskInstance/Details.tsx

```diff
@@ -53,6 +53,7 @@ const Details = ({ gridInstance, taskInstance, group }: Props) => {
   const endDate = gridInstance?.endDate || taskInstance?.endDate;
   const taskId = gridInstance?.taskId || taskInstance?.taskId;
   const mapIndex = gridInstance?.mapIndex || taskInstance?.mapIndex;
+  const executor = taskInstance?.executor || "<default>";
```

Comment:
Just curious about how the default value here. How is that rendered?

Reply:
I believe it's more for displaying, so `<default>` looks good to me.

Reply:
Ah that makes sense. I agree, `<default>` is better from a user's perspective. Thanks!